usertickets/UserTicketSystem/users/models.py

```diff
@@ -1,12 +1,15 @@
 from django.db import models
 from django.contrib.auth.models import AbstractUser
-
+from django.contrib.auth import get_user_model
 class CustomUser(AbstractUser):
     pass
 
-class Ticket(models.Model):  
+class Ticket(models.Model):
     title = models.CharField(max_length=100)
     description = models.TextField()
+    user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)

```

Comment:
what does get_user_model function do?

Reply:
foregnkey establish relation bwtween user and ticket. user model shows which user is active